### ISSUE TITLE
connection.whois() does not take single strings

### ISSUE THREAD

**Issue by jaraco** [jaraco/irc#46]
The connection.whois() method should check to see whether the parameter it was passed was a string or a list and act accordingly.

With a string:

```
#!python
>>> "WHOIS " + ",".join( "JoshTheEnder" ) 
'WHOIS J,o,s,h,T,h,e,E,n,d,e,r' 
```

With a list (which i assume is how it's supposed to be done but seems pointless for 1 user):

```
#!python
>>> "WHOIS " + ",".join( ["JoshTheEnder"] )
'WHOIS JoshTheEnder' 
```

Suggestion: Making the function check the type of the argument and act accordingly between string and list objects

---
- Bitbucket: https://bitbucket.org/jaraco/irc/issue/46
- Originally reported by: Lizzy Trickster
- Originally created at: 2014-10-21T23:54:14.277

**Comment by jaraco** [comment]
Good idea.

---

Original comment by: Jason R. Coombs

**Comment by jaraco** [comment]
Use always_iterable on whois command. Fixes #46.

→ <<cset 14e64cfc83c2>>

---

Original comment by: Jason R. Coombs
